app/controllers/admin/loans_controller.rb

```diff
@@ -1,5 +1,17 @@
 class Admin::LoansController < Admin::AdminController
   def index
     @loans_grid = initialize_grid(Loan, include: [:division, :organization])
+
+    # TODO:
+      # Only load coops/organizations with loans into loan grid
+      # Filter in view should only show coops with loans
```

Comment:
I'm not sure I agree that we should only show co-ops with loans. It might be desirable to be able to select a co-op and see "No loans" rather than searching for it and wondering why it's missing.